### DISCUSSION TITLE
How can I test Venmo flow

### DISCUSSION THREAD

**Discussion by akashbhardwaj** [braintree/braintree_ios#1068] [Q&A]
I would like to test a successful deposit flow with Venmo by launching Venmo app. I don't have an account cannot create one. Is there a way to test the whole flow ? I also went through this page https://developer.paypal.com/braintree/docs/guides/venmo/testing-go-live

**Comment by scannillo** [comment]
Please consult our [Developer Documentation ](https://developer.paypal.com/braintree/docs/guides/venmo/overview)that covers how to build and test a Venmo integration.

For help troubleshooting your specific integration, reach out to[ Braintree Support](https://developer.paypal.com/braintree/help). Please open a [GitHub issue](https://github.com/braintree/braintree_ios/issues/new/choose) if you've found an issue or bug with our SDK.
